### ISSUE TITLE
Error with vanilla installation

### ISSUE THREAD

**Issue by hurrtz** [mxstbr/react-boilerplate#1246]
# React Boilerplate

## Issue Type

- [x] Bug
- [ ] Feature

## Description

With installing the vanilla boilerplate, not doing anything with it and running `npm run build` I get the following error message:

`ERROR in ./~/react-addons-test-utils/index.js
Module not found: Error: Can't resolve 'react-dom/lib/ReactTestUtils' in '/Users/winkler/Sites/chat/node_modules/react-addons-test-utils'
 @ ./~/react-addons-test-utils/index.js 1:17-56
 @ ./~/react-element-to-jsx-string/index-dist.js
 @ ./~/chai-enzyme/build/assertions/contain.js
 @ ./~/chai-enzyme/build/index.js
 @ ./internals/testing/test-bundler.js`

## Steps to reproduce

`git clone --depth=1 https://github.com/mxstbr/react-boilerplate.git .`
`npm run setup`
`npm run build`

# Versions

- Node/NPM: 7.00/4.0.1

**Comment by Dattaya** [comment]
It seems I've fixed the dev branch in #1248, but you have this same bug in the master, right?

**Comment by prasadsilva** [comment]
My react-boilerplate app suddenly started failing in CI, and it was related to react 15.4 getting included somewhere down in the npm dependencies. I fixed this by locking down the semver for all deps/dev-deps (remove `^`) and `npm i --save-dev react-addons-test-utils@15.3` (version depends on when you cloned this repo).

Is there a better solution?

**Comment by Dattaya** [comment]
@prasadsilva, yarn positions itself as a better alternative to `npm` + `npm shrinkwrap`, a quote from https://yarnpkg.com/en/docs/yarn-lock:

> It’s similar to npm’s npm-shrinkwrap.json, however it’s not lossy and it creates reproducible results.

**Comment by gihrig** [comment]
React-boilerplate ships with yarn.lock. In dev branch that installs 15.3.2.

```
"react-addons-test-utils@^0.14.8 || ^15.0.1":
  version "15.3.2"
  resolved "https://registry.yarnpkg.com/react-addons-test-utils/-/react-addons-test-utils-15.3.2.tgz#c09a44f583425a4a9c1b38444d7a6c3e6f0f41f6"
```

**Comment by hurrtz** [comment]
@Dattaya Yes, I am using the default master branch. Thanks for your work so far! I am curious to see if you hotfix this or include test-utils as a dependency :)

**Comment by gihrig** [comment]
The solution for 3.3.3 is to install with `yarn install` rather than `npm run setup` or `npm install`.

This issue will be resolved in 3.3.4 master.

Closing as resolved, but feel free to comment here if desired.
